sql/patch.sql

```diff
@@ -147,3 +147,106 @@ KEY (`success`)
 ) ENGINE=InnoDB;
 #EndIf
 
```

Comment:
for the document_templates and document_template_profiles could simplify by just using same tables from database.sql (assuming those are the most updated tables) and not need to then also then modify them later on in this script